services/ui-src/src/utils/reports/entities.ts

```diff
@@ -56,3 +64,70 @@ export const entityWasUpdated = (
   originalEntity: EntityShape,
   newEntity: AnyObject
 ) => JSON.stringify(originalEntity) !== JSON.stringify(newEntity);
+
+// rendering dynamic Target Population fields on the WP and SAR forms
```

Comment:
Would love suggestions on how to make this one a bit more dynamic, this feels like it's a bit hack-y.

Reply:
(Unless it's not and we just keep it)